lib/angular_wait_barrier.ts

```diff
@@ -0,0 +1,106 @@
+import {SimpleWebDriverClient} from './simple_webdriver_client';
+import {WebDriverCommand} from './webdriver_commands';
+import {WebDriverLogger} from './webdriver_logger';
+import {WebDriverBarrier} from './webdriver_proxy';
+
+let angularWaits = require('./client_scripts/wait.js');
+
+export class AngularWaitBarrer implements WebDriverBarrier {
+  // The ng-app root to use when waiting on the client.
+  rootSelector: string;
+  enabled: boolean;
+  logger: WebDriverLogger;
+
+  constructor(private client: SimpleWebDriverClient) {
+    this.enabled = true;
+    this.rootSelector = '';
+  }
+
+  setRootSelector(selector: string) {
```

Comment:
Should have a JSDoc comment block when `setRootSelector` how it is used or what it is doing.